src/Mod/Path/App/AreaPyImp.cpp

```diff
@@ -160,7 +160,7 @@ static const PyMethodDef areaOverrides[] = {
     {
         "abort",(PyCFunction)areaAbort, METH_VARARGS|METH_KEYWORDS|METH_STATIC,
         "abort(aborting=True): Static method to abort any ongoing operation\n"
-        "\nTo ensure no stray abortion is left in the previous operaion, it is advised to manually clear\n"
+        "\nTo ensure no stray abortion is left in the previous operation, it is advised to manually clear\n"

```

Comment:
`abortion` sounds really _off_ but I looked it up and one of it's several meanings is:  
`arrest of development (as of a part or process) resulting in imperfection; also :  a result of such arrest`  
So technically it is correct. Does anyone have any input ?